airbyte-integrations/connectors/destination-snowflake/metadata.yaml

```
@@ -5,7 +5,7 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 424892c4-daac-4491-b35d-c6688ba547ba
-  dockerImageTag: 3.15.11-rc.1
+  dockerImageTag: 3.15.12
```

we should keep this on an RC tag for now (this is to do a progressive rollout, rather than releasing the connector to everyone)
```suggestion
  dockerImageTag: 3.15.11-rc.2
```